ui/utils/utils.js

```diff
@@ -99,18 +99,24 @@ export function getComponentsinFile(file) {
 
 export function generateValidatePayload(pattern_file, workloadTraitSet) {
   let pattern = jsYaml.loadAll(pattern_file)
+  console.log("Pattern",pattern);
   const services = pattern[0]?.services;
   if (!services) {
     return { err : "Services not found in the design" };
   }
 
   const validationPayloads = {};
-
-  for (const serviceId in services) {
+  console.log("Services",services);
```

Comment:
console.debug here would be good.

Reply:
I would remove this code and might file another pr as I tried to address another issue [6715](https://github.com/meshery/meshery/issues/6715)